.github/workflows/bindings.yml

```diff
@@ -73,6 +73,7 @@ jobs:
         with:
           submodules: recursive
       - uses: ilammy/msvc-dev-cmd@v1
+      - uses: rui314/setup-mold@v1
```

Comment:
Why add the mold linker here in a completely unrelated PR?
IMHO this should happen in a separate PR so it automatically gets a separate Changelog entry.

Reply:
Good point, I'll remove it from this PR.